Mathlib/Analysis/Fourier/LpSpace.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2025 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+module
+
+public import Mathlib.Analysis.Distribution.FourierSchwartz
+public import Mathlib.Analysis.Normed.Operator.Extend
+
+/-!
+
+# The Fourier transform on L^p
+
+In this file we define the Fourier transform on `L2` as a linear isometry equivalence.
```

Comment:
Since `L2` isn't really a thing on its own. Likewise elsewhere.
```suggestion
In this file we define the Fourier transform on $L^2$ as a linear isometry equivalence.
```